src/main/java/de/doubleslash/keeptime/model/repos/UserRepository.java

```diff
@@ -0,0 +1,10 @@
+package de.doubleslash.keeptime.model.repos;
+
+import de.doubleslash.keeptime.model.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface UserRepository extends JpaRepository<User, String> {
```

Comment:
Do we need both, UserRepository and AuthorityRepository? I did not validate this but I assume we only need one table to store the users?